app/models/manifest.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../search/query_dsl'
```

Comment:
I might be wrong, but I think everything in `app` is autoloaded so this shouldn't be necessary